dist/sql-formatter.js

```diff
@@ -516,7 +522,7 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	        this.WHITESPACE_REGEX = /^(\s+)/;
 	        this.NUMBER_REGEX = /^((-\s*)?[0-9]+(\.[0-9]+)?|0x[0-9a-fA-F]+|0b[01]+)\b/;
-	        this.OPERATOR_REGEX = /^(!=|<>|==|<=|>=|!<|!>|\|\||::|->>|->|\{\{\{|\}\}\}|\{\{#\w+\s*\}\}|\{\{|\}\}|.)/;
+	        this.OPERATOR_REGEX = /^(!=|<>|==|<=|>=|!<|!>|\|\||::|->>|->|\{\{\{|\}\}\}|\{\{|\}\}|.)/;
```

Comment:
This will break 2.0 code such as  `{#if`

Reply:
Fixed nha anh